src/app/pages/private/profile/change-password/change-password.page.ts

```diff
@@ -40,18 +31,20 @@ import { ChangePasswordForm } from './change-password.models';
     ReactiveFormsModule,
     TranslateModule,
     HeaderComponent,
+    BackButtonComponent,
   ],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export default class ChangePasswordPage {
   private readonly formBuilder = inject(NonNullableFormBuilder);
-  private readonly navController = inject(NavController);
   private readonly usersApiService = inject(UsersApiService);
   private readonly loaderService = inject(LoaderService);
   private readonly toastService = inject(ToastService);
   private readonly translateService = inject(TranslateService);
   private readonly cdr = inject(ChangeDetectorRef);
 
+  readonly profilePath: string = '/profile';
```

Comment:
not sure if we need to save it to variable. If we want to keep routes as constants - we ave to create global object with all the routes. Based on context in template it will be clear enough that this string is a route path

Reply:
```html
    <div class="tw-p-4">
      <app-back-button [routePath]="'/profile'" />
```